less/variables.less

```diff
@@ -0,0 +1,2 @@
+@titleFont:'Josefin Sans', serif;
+@headercolor:#dab3bc;
```

Comment:
Great variables for a start! Eventually, we'll be adding all sorts of variables (and that's if you decide you want to keep using LESS)